networks.py

```diff
@@ -185,19 +185,25 @@ def _get_uids(people, upper_age=None, by_sex=True):
     def set_risk_groups(self, people, upper_age=None):
         """ Assign each person to a risk group """
         f_uids, m_uids = self._get_uids(people, upper_age=upper_age)
-        self.risk_group[f_uids] = self.pars.risk_groups_f.rvs(f_uids) - f_uids
-        self.risk_group[m_uids] = self.pars.risk_groups_m.rvs(m_uids) - m_uids
+        self.risk_group[f_uids] = self.pars.risk_groups_f.rvs(f_uids)
+        self.risk_group[m_uids] = self.pars.risk_groups_m.rvs(m_uids)

```

Comment:
Am pretty sure that whereas `- uids` was needed before, that was due to a bug which has now been fixed